### DISCUSSION TITLE
I'm unstarring repos thanks to the new dashboard

### DISCUSSION THREAD

**Discussion by groue** [community/community#67243] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Do I need to know about all those releases? Maybe not. It's time unstar those repos, after all. Thanks to the new dashboard, I can clean up my star history.

EDIT: hell, the new dashboard still shows releases about those repos I have unstarred 😖 What a poor, unfinished, not sensible  update. Is there anything well executed here?

**Comment by leeoniya** [comment]
> Is there anything well executed here?

the utility of the feed was executed -- like by hanging or lethal injection.

**Reply by ascherer** [reply]
I unstarred all repos when the "create your first list" bs appeared.

**Comment by ShaikhZayan** [comment]
res 1
